Makefile

```diff
@@ -0,0 +1,43 @@
+DOCKER_IMAGE_NAME = localhost/galaxy_ng/galaxy_ng
+
+
+.PHONY: help
+help:
+	@echo "Usage: make <target>"
+	@echo ""
+	@echo "Targets:"
+	@echo "requirements                Update python dependencies lock files (i.e. requirements.txt)."
+	@echo ""
+	@echo "Docker targets:"
+	@echo "docker/build                Build all development images."
+	@echo "docker/build/standalone     Build image for standalone environment."
+	@echo "docker/build/insights       Build image for insights environment."
+
+
+.PHONY: requirements
+requirements:
+	pip-compile -U -o requirements/requirements.common.txt \
+		setup.py
+	pip-compile -U -o requirements/requirements.standalone.txt \
+		setup.py requirements/requirements.standalone.in
+	pip-compile -U -o requirements/requirements.insights.txt \
+		setup.py requirements/requirements.insights.in
+
+
+.PHONY: docker/build
+docker/build: docker/build/insights docker/build/standalone
```

Comment:
This will make it so that you have to run `make docker/build` to build the API and `./compose.sh build` to build the UI.

Is it it possible to add `./compose.sh build` in here as well?